bin/skonfig

```diff
@@ -1,18 +1,30 @@
 #!/usr/bin/env python3
 
+def run():
+    import skonfig.cdist
+    if os.path.basename(sys.argv[0])[:2] == "__":
+        return skonfig.cdist.emulator()
+    else:
+        import skonfig.arguments
+        arguments = skonfig.arguments.get()
+        if arguments.dump:
+            import skonfig.dump
+            return skonfig.dump.run(arguments.host)
+        else:
+            return skonfig.cdist.run(arguments)
+
+
 if __name__ == "__main__":
     import os
-    import sys
-
     skonfig_bindir_pardir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
-
+    import sys
     if os.path.isfile(os.path.join(skonfig_bindir_pardir, "skonfig", "__init__.py")):
         sys.path.insert(0, skonfig_bindir_pardir)
-
     import skonfig
     skonfig.THIS_IS_SKONFIG = True
-
     try:
-        sys.exit(skonfig.run())
+        if run():
```

Comment:
I think it would be nice if the `run()` function could return an exit code instead of a `bool`.
Then this `if` could be simplified to `sys.exit(run())`.